tilesets/generate_tiles.py

```
@@ -136,6 +149,12 @@ def generate_1d_tiles(filename, tile_ids, get_data_function):
 
         dense = get_data_function(filename, tile_position)
 
+        if agg_info != None:
+            agg_func_name = agg_info["agg_func"]
+            agg_group_arr = [ x if type(x) == list else [x] for x in agg_info["agg_groups"] ]
+            assert(agg_func_name in agg_func_map)
```

Might be good to return some meaningful error message here. Maybe raise an exception and then catch it later and return as `{"error": message}` in the tile return value.